R/rank_list.R

```diff
@@ -72,25 +77,37 @@ rank_list <- function(
     NULL
   }
 
-  z <- tagList(
+  label_tags <- mapply(
+    USE.NAMES = FALSE,
+    SIMPLIFY = FALSE,
+    labels,
+    label_ids(labels),
+    FUN = function(label, label_id) {
+      tags$div(
+        class = "rank-list-item",
+        "data-rank-id" = label_id,
```

Comment:
If `label_id` is `""`, put `NULL` here so that the attribute doesn't show up at all in the tag.